doc/advanced-configuration/Modules.md

```diff
@@ -84,17 +106,22 @@ This module implements [XEP-0016: Privacy Lists)](http://xmpp.org/extensions/xep
 ### [mod_private](../modules/mod_private.md)
 Implements [XEP-0049 (Private XML Storage)](http://xmpp.org/extensions/xep-0049.html) to store and query private user data in XML format.
 
+### [mod_pubsub](../modules/mod_pubsub.md)
+This extension implements [XEP-0060 (Publish-Subscribe)](http://www.xmpp.org/extensions/xep-0060.html). It is a pluggable implementation, that uses behaviours provided by `node_*.erl` and `nodetree_*.erl` modules.
```

Comment:
...a pluggable implementation using behaviours...

Reply:
both work, I slightly lean toward "that"